malasakit-django/locale/tl/LC_MESSAGES/django.po

```diff
@@ -148,39 +156,42 @@ msgstr "Simulan"
 
 #: pcari/templates/peer-responses.html:5 pcari/templates/peer-responses.html:6
 msgid "Peer Responses"
-msgstr ""
+msgstr "Tugon ng iba"
 
 #: pcari/templates/peer-responses.html:32
 msgid ""
 "Below, you can see how the other respondents answered the quantitative "
 "questions."
 msgstr ""
+"Makikita sa ibaba kung paano sinagutan ng iba ang mga kuwantitatibong katanungan."
 
 #: pcari/templates/peer-responses.html:42
 #, python-format
 msgid "<strong>%(num_ratings)s</strong> other people answered this question:"
-msgstr ""
+msgstr "<strong>%(num_ratings)</strong> katao na ang sumagot sa tanong na ito:"
 
 #: pcari/templates/peer-responses.html:50
 #, python-format
 msgid "Your answer: <strong id=\"answer-%(id)s\">None</strong>"
-msgstr ""
+msgstr "Ang iyong sagot: <strong id=\"sagot-%(id)\">Wala</strong>"
```

Comment:
The HTML code inside the pairs of angle brackets (`<...>`) should not be translated, so please change `sagot-%(id)` back to `answer-%(id)s`.